src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts

```diff
@@ -356,17 +392,57 @@ export class GettingStartedPage extends EditorPane {
 			this.editorInput.selectedTask = id;
 			this.selectedTaskElement = taskElement;
 			const taskToExpand = assertIsDefined(this.currentCategory.content.items.find(task => task.id === id));
+			if (taskToExpand.media.type === 'image') {
+
+				this.taskMediaComponent.classList.add('image');
+				this.taskMediaComponent.classList.remove('markdown');
+
+				const media = taskToExpand.media;
+				const mediaElement = $<HTMLImageElement>('img');
+				this.taskMediaComponent.appendChild(mediaElement);
+				mediaElement.setAttribute('alt', media.altText);
+				this.updateMediaSourceForColorMode(mediaElement, media.path);
+
+				this.taskDisposables.add(addDisposableListener(mediaElement, 'load', () => mediaElement.width = mediaElement.naturalWidth * 2 / 3));
+				this.taskDisposables.add(this.themeService.onDidColorThemeChange(() => this.updateMediaSourceForColorMode(mediaElement, media.path)));
+
+			} else if (taskToExpand.media.type === 'markdown') {
+
+				this.taskMediaComponent.classList.remove('image');
+				this.taskMediaComponent.classList.add('markdown');
+
+				const media = taskToExpand.media;
+
+				const webview = this.taskDisposables.add(this.webviewService.createWebviewOverlay(this.webviewID, {}, {
```

Comment:
Could you use a normal webview element instead of an overlay?

The overlay is only for cases where we need to make sure the webview is never reparented. Since the webview here does not have any state, I think it may be ok to let it be reparented

Reply:
Ah, good tip thanks, cleaned up some code.